pkg/mcs/tso/server/apis/v1/api.go

```diff
@@ -185,6 +185,11 @@ func ResetTS(c *gin.Context) {
 	if err = svr.ResetTS(ts, ignoreSmaller, skipUpperBoundCheck, 0); err != nil {
 		if err == errs.ErrServerNotStarted {
 			c.String(http.StatusInternalServerError, err.Error())
+		} else if err == errs.ErrEtcdTxnConflict {
```

Comment:
This is not only for test? Because it will effect the resp to users.
I want to know whether this PR needs to cherry-pick.

Reply:
yes, not only for test. But I am not sure whether it is a bug to need cherry-pick.

Reply:
If it's only for TSO services, IMO we don't need to.

Reply:
PD mode also returns 403 when meeting ErrEtcdTxnConflict

Reply:
BR is using it https://github.com/pingcap/tidb/blob/1f7c1e02bcd402cfbc1a0b0a26acfbd0d0fe4d8c/br/pkg/pdutil/pd.go#L892

Reply:
After discussion with @niubell, we will only fix it on master branch.